Felienne/hedy
# [BUG FIX] syntax highlighting works again on public profile

**Description**
This is a very small fix where we fix a forgotten change on the level attribute for the ace editor. Previously we used the `id` of a `pre` element to indicate the Hedy level for which we needed syntax highlighting. As this is a bit against the HTML best-practices we changed this to added a `level` attribute to the element. However, this change was forgotten on the public profile page.

**Fixes**
This PR fixes #2214

**How to test**
Navigate to a public profile with programs and verify that the syntax highlighting is working.
